### ISSUE TITLE
Wrong CRS defintion in spatial_ref_sys.txt

### ISSUE THREAD

**Issue by stefan0722** [GeoLatte/geolatte-geom#41]
If you call crsMap.get(3857) in CrsRegistry you get the CRS 3785 definition which is different. When using with hibernate spatial, the geometry will be persisted in a different SRID which leads to a inconsistent db state.

A workaround is to shadow the spatial_ref_sys.txt and exchange the PROJCS definition with the one from http://spatialreference.org/ref/sr-org/7483/postgis/.

**Comment by thekalinga** [comment]
Any plan on fixing this. This is a blocker.

**Comment by thekalinga** [comment]
@stefan0722 Would you mind specifying why you have pointed to the definition of SRID 7483 instead of 3857?

Should we this instead SRID 3837 [WKT](http://epsg.io/3857.wkt) or [Post GIS version](http://epsg.io/3857.sql)?

Thanks

**Comment by thekalinga** [comment]
@stefan0722 I got it now, the reference number 7483 is from spatialreference.org number, not the epsg :)

**Comment by kozyr** [comment]
This makes it impossible to use Hibernate Spatial 5.0+...

**Comment by maesenka** [comment]
The fix is now released in version 1.0.3

**Comment by thekalinga** [comment]
Have the following issue with 1.0.3

```
[2016-07-28 17:38:07.165] log4j2 - 15965  WARN [http-nio-8080-exec-2] --- CrsRegistry: Can't parse srid 3857 ('PROJCS["WGS 84 / Pseudo-Mercator",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Mercator_1SP"],PARAMETER["central_meridian",0],PARAMETER["scale_factor",1],PARAMETER["false_easting",0],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["X",EAST],AXIS["Y",NORTH], AUTHORITY["EPSG","3857"]]'). 
Illegal Character at pos 0 in Wkt text: 'PROJCS["WGS 84 / Pseudo-Mercator",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Mercator_1SP"],PARAMETER["central_meridian",0],PARAMETER["scale_factor",1],PARAMETER["false_easting",0],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["X",EAST],AXIS["Y",NORTH], AUTHORITY["EPSG","3857"]]'
```
